### PR TITLE
Lats

### PR DESCRIPTION
**Pull request by doutriaux1**
- c/c++ `@conda-forge/help-c-cpp`

### CODE REVIEW

**Comment by doutriaux1** [comment]
@conda-forge/help-c-cpp this is ready for review

File: recipes/lats/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+build:
+  number: 0
+  skip: True  # [win]
+ 
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('fortran') }}
+  host:
+    - libnetcdf 4.*
```
Comment (mariusvniekerk) [inline recipes/lats/meta.yaml]: could we rely on the pinnings provided by conda-forge

**Comment by doutriaux1** [comment]
@mariusvniekerk the original recipe that used to pass now fails. Any idea what happened? I don't think it's related to my recipe.

**Comment by doutriaux1** [comment]
@mariusvniekerk i need help here please.

**Comment by mariusvniekerk** [comment]
i'd rebase this on the current master, we made some pretty big changes to staged recipes recently

**Comment by doutriaux1** [comment]
@mariusvniekerk ok but apparently your changes led to the build failing 😄 will try again...

**Comment by doutriaux1** [comment]
Apparently it is up to date with master...

**Comment by durack1** [comment]
@doutriaux1 Does that mean lats is now part of conda-forge?

**Comment by doutriaux1** [comment]
It means it never passed and went dead. “someone” should look at it again and get it going again

C.

**Pull request closed without merging**